package.json

```diff
@@ -24,11 +28,24 @@
   "homepage": "https://github.com/windbreaker-io/npm-watcher#readme",
   "devDependencies": {
     "ava": "^0.19.1",
-    "eslint": "^4.0.0"
+    "eslint": "^4.0.0",
+    "eslint-config-standard": "^10.2.1",
+    "eslint-plugin-import": "^2.3.0",
+    "eslint-plugin-node": "^5.0.0",
+    "eslint-plugin-promise": "^3.5.0",
+    "eslint-plugin-standard": "^3.0.1",
+    "nyc": "^11.0.2"
   },
   "dependencies": {
     "amqplib": "^0.5.1",
     "bluebird": "^3.5.0",
+    "changes-stream": "^2.2.0",
+    "fashion-model-defaults": "^1.0.1",
+    "normalize-registry-metadata": "^1.1.2",
+    "redis": "^2.7.1",
```

Comment:
You should use [`ioredis`](https://github.com/luin/ioredis) instead. It has great promise functionality built in, great support for msgpack, and redis cluster.